app/views/layouts/application.html.erb

```diff
@@ -1,131 +1,208 @@
 <!doctype html>
-
-<html lang="en">
+<!-- 
+Karla Rodriguez 
+added localization related meta tags from style guide
+-->
+<!--[if lt IE 9]><html class="lt-ie9"><![endif]-->
+<!--[if gt IE 8]><!--><html lang="en"><!--<![endif]-->
   <head>
     <meta charset="utf-8">
+<!-- 
+Karla Rodriguez 
+added mobile related meta tags from style guide
+-->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="HandheldFriendly" content="True">
+    <meta name="MobileOptimized" content="320">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    
+
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title><%= Rails.application.config.app_name %></title>
-    <link rel="stylesheet" href="/assets/uswds/css/uswds.min.css">
+    <title>login.gov Dashboard</title>
+    
+<!-- 
+Karla Rodriguez 
+removed link to uswds.min.css, added shortcut icons and added link to style guide in node modules 
+-->
+    <!-- 128x128 -->
+    <link rel="shortcut icon" type="image/ico" href="/assets/img/favicons/favicon.ico">
+    <link rel="icon" type="image/png" href="/assets/img/favicons/favicon.png">
```

Comment:
I'm looking at icons throughout and getting them to work in production with compile assets.

Reply:
👍🏼